invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -366,6 +366,7 @@ export class RDMDepositForm extends Component {
                       fieldPath="metadata.subjects"
                       initialOptions={_get(record, "ui.subjects", null)}
                       limitToOptions={this.vocabularies.metadata.subjects.limit_to}
+                      searchOnFocus={this.vocabularies.metadata.subjects.search_on_focus}
```

Comment:
```suggestion
                      searchOnFocus={true}
```
I think adding a backend config variable is a bit an overkill, we can control this entirely on the frontend

Reply:
This switch won't be as accessible in that case. Also if we want to add a new functionality similar to this, it would be easy with the config. Especially for the inbuilt props which could be passed to the config without updating the component. WDYT?

Reply:
lets dicuscuss IRL, I am not sure I understand what you mean so let's clarify